data/coalition/coalition outfits.txt

```diff
@@ -239,15 +239,15 @@ outfit "Refueling Module"
 
 outfit "Shield Refactor Module"
 	category "Systems"
-	cost 2843000
+	cost 8324000
 	thumbnail "outfit/shield refactor module"
-	"mass" 7
-	"outfit space" -7
-	"hull repair rate" -.2
+	"mass" 11
+	"outfit space" -11
+	"hull repair rate" -.25
 	"shield energy" .24
 	"shield heat" .32
```

Comment:
```suggestion
	"shield energy multiplier" .2
	"shield heat" .32
```
I think that multiplicative energy + flat heat fits the best here. Based on the description, what's happening is the shields are being operated at a higher level, and some hull repair is being diverted to repair them. So, you're running the shields proportionally at a higher load, and hence it makes sense that an equally proportional higher energy draw is taken.
Shield heat, on the other hand, is 0 for most shield generators, and makes sense to be less directly and proportionally linked to the increased generation, so I think a flat addon to heat makes more sense here than a multiplier.